src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const todoList = {
+  id: "First",
+  title: "Review notes",
+  id: "Second",
+  title: "Listen to next 2 React sections",
+  id: "Third",
+  title: "Make dinner",
+}
+
+function App() {
+  <div>
+  <h1>Todo List</h1>
+  <ul>{todoList.map}</ul>
```

Comment:
The `.map()` function receives a [Callback function](https://developer.mozilla.org/en-US/docs/Glossary/Callback_function) as an argument. 

It is within that callback function that we will return our `<li>` elements!

You can read more about `.map()` on [MDN docs](**url**).